submissions/yhorodechna/frogger-oop/app.js

```diff
@@ -0,0 +1,156 @@
+const SCORE = 0;
+const CANVAS = {
+    width: 505,
+    height: 605,
+    cellHeight: 83,
+    cellWidth: 101,
+};
+/* The function RESET_ENEMIES_SPEED add for each enemy random speed. This function is used in class Enemy 
+to reset the speed of each enemy when a player loses */
+const RESET_ENEMIES_SPEED = function () {
+    for (let i = 0; i < allEnemies.length; i++) {
+        allEnemies[i].speed = Math.floor(Math.random() * (100 - 40)) + 40;
+    }
+};
+class Enemy {
```

Comment:
As we go for OOP `class` syntax, then make the code OOP-compliant, including generalization of shared props and their handling into a base class.
OOP is not just about syntax, it is more about principles that help building extensible and maintainable code.

Reply:
I created  new class that extends Player and Enemy classes